condicionales/numeroParOImpar.js

```diff
@@ -1,27 +1,17 @@
 // Escriba un programa que pida primero un número par y luego un número impar (positivos o negativos). En caso de que uno o los dos valores no sea correcto, se mostrará un único aviso.
 const prompt = require("prompt-sync")();
 
-let numberPair;
+let numberEven;
 let numberOdd;
 
-do {
-    numberPair = parseInt(prompt("Enter a number Pair: "));
-  
-    if (isNaN(numberPair)) {
-      console.log("Error: Enter a valid number!");
-    }
-  } while (isNaN(numberPair));
-  do {
-    numberOdd = parseInt(prompt("Enter a number Odd: "));
-  
-    if (isNaN(numberOdd)) {
-      console.log("Error: Enter a valid number!");
-    }
-  } while (isNaN(numberOdd));
+numberEven = parseInt(prompt("Enter a number Pair: "));
```

Comment:
Maybe if you use "parseInt" is going to work not like expected, for example, if the user enters 4.3, is going to parseInt to 4, and is Even, but 4.3 is not right